urllib3/contrib/appengine.py

```diff
@@ -85,9 +101,25 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         # Return False to re-raise any potential exceptions
         return False
 
-    def urlopen(self, method, url, body=None, headers=None,
-                retries=None, redirect=True, timeout=Timeout.DEFAULT_TIMEOUT,
-                **response_kw):
+    def clear(self):
+        pass
+
+    def connection_from_url(self, url):
+        return _AppEngineConnection(self, url)
+
+    def urlopen(self, method, url, body=None, headers=None, retries=None,
+                redirect=True, assert_same_host=True, timeout=Timeout.DEFAULT_TIMEOUT,
+                pool_timeout=None, release_conn=None, **response_kw):
+        """This should be an AppEngine version of HTTPConnectionPool.urlopen().
```

Comment:
"This should be an " prefix doesnt appear to add much value.

"AppEngine version of HTTPConnectionPool.urlopen()." says the same?

Or pep257 compliant would be something like

"Perform an HTTP request using AppEngine's URLFetch."